akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/StrictTransportSecurityDirective.scala

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright (C) 2009-2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.http.scaladsl.model.headers
+
+sealed abstract class StrictTransportSecurityDirective
+case class IgnoredDirective(value: String) extends StrictTransportSecurityDirective
```

Comment:
Make the case classes final to protect from user extension (makes bincomp evolution easier)